src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+public class TaskList {
+    /** Array of tasks */
+    private Task[] tasks;
+
+    /** Number of tasks */
+    private int noOfTasks = 0;
+    public TaskList(int taskLength) {
+        tasks = new Task[taskLength];
+    }
+
+    /**
+     * Adds task to the task list
+     * @param task The task
+     */
+    public void addTask(Task task){
+        tasks[noOfTasks] = task;
+        noOfTasks++;
+    }
+
+    /**
+     * Checks the task in the task list
+     * @param taskNumber The index of the task
+     */
+    public void checkTask(int taskNumber){
+        tasks[taskNumber-1].setTaskStatus(true);
+        System.out.println("      " + tasks[taskNumber-1]);

```

Comment:
Perhaps you can use other printing characters for the spaces? I suggest `\t` char for tabs